src/Tabs/Tabs.js

```diff
@@ -202,6 +207,7 @@ class Tabs extends Component {
         createElement(tabTemplate || TabTemplate, {
           key: index,
           selected: this.getSelected(tab, index),
+          templateStyles: tabTemplateStyle,
```

Comment:
TabTemplate already has a `styles` variable, I don't think that it would look good to have `styles` and `style`.

What's your opinion on this?